docs/downstream/crc.md

```diff
@@ -0,0 +1,34 @@
+# Deployment on CRC
+
+These is how to deploy ArgoCD on [CRC](https://developers.redhat.com/products/codeready-containers/overview).
+
+## Instalation Steps
+
+ * Setup CRC https://developers.redhat.com/products/codeready-containers/overview
+   * Do not forget to install the corresponding version of `oc` tool or some commands might fail.
+   * add more memory to CRC : \
+   `crc delete` \
+   `crc config set memory 16384` \
+   `crc start`
+
+ * Get `kustomize` and KSOPS using steps in [manage_your_app_secrets.md](../manage_your_app_secrets.md)
+
+   * If you hit:\
+``` unrecognized import path "vbom.ml/util": https fetch: Get "https://vbom.ml/util?go-get=1": dial tcp ```\
+fix it with https://github.com/viaduct-ai/kustomize-sops/issues/60
+
+* As an alternative to installing the prerequisites locally, you can use Toolbox: https://github.com/containers/toolbox \
+   `toolbox create --image quay.io/aicoe/of-toolbox:v0.1.0` \
```

Comment:
I would remove the instructions here on how to use the toolbox since they are mentioned in the link we'd need to make changes to multiple files if the versions change otherwise :)

Reply:
This was part of another PR and is here because of the rebase. let's deal with it separately if needed.